src/kili/mutations/plugins/__init__.py

```diff
@@ -59,6 +60,64 @@ def upload_plugin(
 
         return PluginUploader(self.auth, plugin_path, plugin_name, verbose).create_plugin()
 
+    @typechecked
+    def create_webhook(
+        self,
+        webhook_url: str,
+        plugin_name: str,
+        header: Optional[str] = None,
+        verbose: bool = True,
+    ):
+        # pylint: disable=line-too-long
+        """Create a webhook linked to kili's event to receive custom events.
+
+        Args:
+            webhook_url: Url base receiving push events from on_submit / on_review events on Kili
+            plugin_name: name of your plugin
+            header: Authorization header to access the routes
+            verbose: If false, minimal logs are displayed
+
+        Returns:
+            A result object which indicates if the mutation was successful,
+                or an error message.
+
+        Examples:
+            >>> kili.create_webhook(webhook_url='https://my-custom-url-publicly-accessible/', plugin_name='my webhook', header='...')
+        """
+
+        return WebhookUploader(
+            self.auth, webhook_url, plugin_name, header, verbose
+        ).create_webhook()
+
+    @typechecked
+    def update_webhook(
+        self,
+        webhook_url: str,
+        plugin_name: str,
+        header: Optional[str] = None,
+        verbose: bool = True,
+    ):
+        # pylint: disable=line-too-long
+        """Create a webhook linked to kili's event to receive custom events.
+
+        Args:
+            webhook_url: Url base receiving push events from on_submit / on_review events on Kili
```

Comment:
I think it would be better to use `new_webhook_url` so that users understand it's the new URL that they're supposed to provide